DataAccessLayer/Realization/HeroService.cs

```diff
@@ -0,0 +1,59 @@
+﻿using DataAccessLayer.Interfaces;
+using DataAccessLayer.Core;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Data.Entity;
+
+namespace DataAccessLayer.Realization
+{
+    public class HeroService : IRepository<Heroes>
+    {
+        private TheWitcherEntities db;
+        public HeroService()
+        {
+            this.db = new TheWitcherEntities();
+        }
+        public void Create(Heroes item)
```

Comment:
It is reasonable to return something as a result. For example Id of created hero. In this way we will know that entity was created.